### DISCUSSION TITLE
Help take down my project

### DISCUSSION THREAD

**Discussion by lethanhdat0102** [community/community#120401] [Projects and Issues]
### Select Topic Area

General

### Feature Area

Projects

### Body

My project is secret content. It was accidentally posted here and someone else cloned it. I want to find a way for that person to take it down. It is confidential and legally related content. please help me. This is his address. I have no way to contact him. https://github.com/frontcover

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by lethanhdat0102** [comment]
Please contact to him

**Comment by IsaacHatton** [comment]
@lethanhdat0102 Please see this GitHub Support article:
https://docs.github.com/en/site-policy/content-removal-policies/guide-to-submitting-a-dmca-takedown-notice
